Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -1317,6 +1317,13 @@ script:
       - sc_task
       - sc_req_item
       - sn_si_incident
+    - auto: PREDEFINED
+      defaultValue: false
+      description: Whether to add the ticket notes and work notes as an entry note in the War Room.
```

Comment:
Recommend:
Whether to add ticket notes and work notes as notes in the War Room.